packages/react-core/src/next/components/Wizard/Wizard.tsx

```diff
@@ -40,14 +39,16 @@ export interface WizardProps extends React.HTMLProps<HTMLDivElement> {
   width?: number | string;
   /** Custom height of the wizard */
   height?: number | string;
-  /** Disables navigation items that haven't been visited. Defaults to false */
-  isStepVisitRequired?: boolean;
-  /** Callback function when a step in the navigation is clicked */
-  onNavByIndex?: WizardNavStepFunction;
-  /** Callback function after next button is clicked */
-  onNext?: WizardNavStepFunction;
-  /** Callback function after back button is clicked */
-  onBack?: WizardNavStepFunction;
+  /** Disables steps that haven't been visited. Defaults to false. */
+  isVisitRequired?: boolean;
+  /** Progressively shows steps, where all steps following the active step are hidden. Defaults to false. */
+  isProgressive?: boolean;
+  /** Callback function when navigating between steps */
+  onStepChange?: (
```

Comment:
@tlabaj can correct me if I'm off, but I think we are trying to standardize all callback prop types exposed to the consumers across the library so that they all pass the event back as the first parameter.

Reply:
That would certainly complicate things if that were true for this case. This function is a consolidation of 3 that were already defined as a type without an event handler bound as the first parameter, so I wonder if this could be handled separately as a part of the broader effort to make this change in PF?

Reply:
Hi @jeffpuzzo we are trying to standardize your callback signatures so that an event is always passed and it is the first parameter.  This was an ask from consumers. Some consumers have to wrap PF components in order to use 3rd party libs like Formik due to our inconsistent/non standard following callbacks.   

I am ok with opening a separate issue to tackle that, but we would want to get that in for our major release.

Reply:
@tlabaj @nicolethoen I ended up just trying to add those event props as a part of this PR. Please have a look when you have a moment.